en/docs/get-started/about-this-release.md

```diff
@@ -398,14 +398,54 @@ The following mediators and artifacts are removed from the ESB runtime in all ve
+
+- **SSE Sink / Source support over http**
+    
+	 SSE is an HTTP-based protocol that allows one-way communication similar to WebHooks, from the server to the client. For more information , see [SSE Server Sink](https://siddhi-io.github.io/siddhi-io-http/api/2.3.3/#sse-server-sink) , [SSE Source](https://siddhi-io.github.io/siddhi-io-http/api/2.3.3/#sse-source).
+	 
+- **WebSub Sink / Source support over http**
+    
+	 Implementation is based on the [WebSub](https://www.w3.org/TR/websub/) protocol specification.WebSubs only allows one-way communication, from a caller web app to a callee web app. For more information , see [WebSub Sink](https://siddhi-io.github.io/siddhi-io-http/api/2.3.3/#websubhub-sink) , [WebSub Source](https://siddhi-io.github.io/siddhi-io-http/api/2.3.3/#websubhub-source).
+
+- **Improved MongoDB Store**
+    
+	 Several improvements are done like allow encryption protocol to be configurable, Object_ID support when joining and other bug fixes. For more information , see [Mongo DB Store](https://siddhi-io.github.io/siddhi-store-mongodb/api/2.1.2/#mongodb-store).
+	 
+- **Improved IO CDC Source**
+    
+	 CDC can be used to capture insert / update / delete actions from databases. Release includes improvement where with single CDC configuration it can handle all three actions. For more information , see [CDC Source](https://siddhi-io.github.io/siddhi-io-cdc/api/2.0.10/).
+	 
+- **Synchronous publishing support for IO Kafka**
+    
+	 IO Kafka allows seamless connection with kafka broker and retrieves the events into the streaming integrator. Release includes synchronous publishing in a more controlled manner and other bug fixes. For more information , see [IO-Kafka](https://github.com/siddhi-io/siddhi-io-kafka).
+	 
+- **Improved JSON Mapper to support null values**
+    
+	 JSON mapper provides capability to retrieve or publish events in JSON format. For more information , see [Map-JSON](https://github.com/siddhi-io/siddhi-map-json).
+	 
+- **Improved KeyValue Mapper to support force casting**
+    
+	 KeyValue mapper provides capability to retrieve or publish events in as key value format. For more information , see [Map-KeyValue](https://siddhi-io.github.io/siddhi-map-keyvalue/api/2.1.1/).
+	 
+- **Improved IO File Sink and Source**
+    
+	 IO File sink / source can be used to read or write into a file. With the new versions it has the support for EOF in other modes, support for file trailers, file writing mechanisms and SFTP support. For more information , see [IO-File](https://siddhi-io.github.io/siddhi-io-file/api/2.0.16/).
+	 
+- **Improved Store RDBMS support**
+    
+	 Store RDBMS can be used to connect with any rdbms type. With the new versions it has the support to add null values to RDBMS tables and improved metrics support. For more information , see [Store RDBMS](https://github.com/siddhi-io/siddhi-store-rdbms).
```

Comment:
Better not refer to Store RDBMS as an object in a sentence...
Is this correct?
"The **Store RDBMS** extension is used to connect with any RDBMS type. The new improvements include the possibility to add null values to RDBMS tables and enhanced metrics."

Reply:
This is a technical word for one of the extensions.  I feel this is fine because user should know that this is technical word for said extension

Reply:
okay.. Just added some minor suggestions.. See commit suggestion below.